### PR TITLE
use plotly for task barcharts instead of custom d3

### PR DESCRIPTION
**Pull request by shapiromatron**
Instead of custom react + d3 barchart, switch to plotly. This net removal of code should simplify maintenance in the future.

Screen Shot 2021-08-25 at 11 57 24 PM

### CODE REVIEW

File: frontend/mgmt/Dashboard/Root.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        if (store.error) {
+            return <GenericError />;
+        }
+
+        if (store.isFetching) {
+            return <Loading />;
+        }
+
+        if (store.tasks === null || store.tasks.length === 0) {
+            return <p>To tasks have yet assigned.</p>;
```
Comment (rabstejnek) [inline frontend/mgmt/Dashboard/Root.js:29]: Reword, maybe "No tasks have been assigned."

**Review by rabstejnek — CHANGES_REQUESTED** [review]
Just one suggestion, the text when there's no tasks to display isn't very clear. Otherwise looks good!

File: frontend/mgmt/Dashboard/Root.js (hunk 1 above)
Comment (shapiromatron) [reply frontend/mgmt/Dashboard/Root.js:29]: doh! will fix

**Comment by shapiromatron** [comment]
@rabstejnek ok, made the change -  if it's ok could you please accept and merge?

**Review by rabstejnek — APPROVED** [review]

**Pull request merged**